ext/ffi_c/extconf.rb

```diff
@@ -8,7 +8,8 @@ def system_libffi_usable?
     # We need pkg_config or ffi.h
     libffi_ok = pkg_config("libffi") ||
         have_header("ffi.h") ||
-        find_header("ffi.h", "/usr/local/include", "/usr/include/ffi")
+        find_header("ffi.h", "/usr/local/include", "/usr/include/ffi") ||
+        (find_header("ffi.h", `xcrun --sdk macosx --show-sdk-path`.strip) rescue false)
```

Comment:
This needs + `/include/ffi`, otherwise it doesn't find it: https://github.com/ffi/ffi/pull/765/checks?check_run_id=546591203
`rescue false` seems brittle to me and might swallow unwanted exceptions.

Reply:
I created https://github.com/ffi/ffi/pull/766 to show what I was thinking.

Reply:
> This needs + /include/ffi

Of course! I added "usr/include/ffi" and it seems to work.

To be honest I like this PR (including the rescue) more than #766 . It calls `xcrun` only when necessary, fits well into the chain, is very simple and doesn't break the installation if anything goes wrong.

Reply:
Indeed, I forgot the `/usr` in front!
As you prefer, I'm thinking just searching `/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/ffi` as in #758 is the simplest now.

I personally dislike any silent `rescue`, as it can easily become a debugging nightmare.